frontend/src/ui/components/organisms/attendance/Attendance.scss

```diff
@@ -31,6 +31,24 @@
     scrollbar-color: $rindus-green $rindus-off-white;
     text-align: center;
   }
+
+  &__loader {
```

Comment:
Here you are repeating the same from here: frontend/src/ui/components/atoms/loader/Loader.scss

You should only override the properties that you want to change.